src/Updates/CreateTokensTable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Statamic\Eloquent\Updates;
```

Comment:
This should probably be `UpdateScripts` instead, to stay inline with Core.

```suggestion
namespace Statamic\Eloquent\UpdateScripts;
```

Reply:
We've used updates in eloquent driver (since before my time), so its consistent internally at least.

Reply:
Oh okay! Didn't realise that. 👍